src/services/ui/src/features/package-actions/legacy-shared-components.tsx

```diff
@@ -66,9 +66,12 @@ export const useIntakePackage = () => {
   };
 };
 
+export const isNewSubmission = () =>
+  window.location.href.includes("new-submission");
+
```

Comment:
this is getting used elsewhere and pertains to all new submissions (renewals and amendments will one day be action/new submission accessible like TEs) -- let's pop this out into `ui > src > utils > location.ts` where we do other window math